.github/workflows/on-push.yml

```diff
@@ -346,7 +346,7 @@ jobs:
       - name: Validate content master paths
         run: |
           source $(poetry env info --path)/bin/activate
-          validate-content-path validate-all content --skip-depth-one-file --skip-depth-one-folder --skip-depth-zero-file --skip-integration-script-file-name --skip-integration-script-file-type --skip-markdown
+          validate-content-path validate-all content --skip-depth-one-file --skip-depth-one-folder --skip-depth-zero-file --skip-integration-script-file-name --skip-integration-script-file-type --skip-markdown --skip-suffix
```

Comment:
why this is needed? for gradual implementation?

Reply:
exactly.